### PR TITLE
Revert to resolve=False to_yaml()

### PR DESCRIPTION
**Pull request by pereman2**
### Jupyter notebook looks the same as before
image

### CODE REVIEW

File: docs/notebook/Tutorial.ipynb [hunk 1]
```diff
@@ -575,47 +570,48 @@
...
+    "# Composite interpolation types are always string\n",
+    "print(\"conf.client.url: \", conf.client.url, type(conf.client.url).__name__)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Notice that the url and server_port changes with the port."
+    "You can resolve interpolation while you are printing using `resolve=True`."
```
Comment (omry) [inline docs/notebook/Tutorial.ipynb:589]:
```suggestion
    "to_yaml will resolve interpolation if `resolve=True` is passed"
```

**Comment by gerardsn** [comment]
the pretty() warning still states that the default has changed to resolve=True

**Comment by omry** [comment]
@gerardsn, thanks for reporting. can you open a new issue or better yet - send a pull request fixing it?

**Comment by omry** [comment]
never mind, I got this.

**Pull request merged**